README.md

```diff
@@ -84,6 +84,7 @@ The following opcodes are supported in the opcode item:
     0x15 - operation: ECDSA sign SHA256
     0x16 - operation: ECDSA sign SHA384
     0x17 - operation: ECDSA sign SHA512
+    0x24 - operation: Custom Function
```

Comment:
OpRPC isn't in this README. Should we add it?

Reply:
Will do.